packages/cli/src/config/auto-init/ApimlAutoInitHandler.ts

```diff
@@ -99,27 +103,55 @@ export default class ApimlAutoInitHandler extends BaseAutoInitHandler {
         }
         const profileConfig = Services.convertApimlProfileInfoToProfileConfig(profileInfos);
 
+        const config = ImperativeConfig.instance.config;
+        // Check to see if there is an active base profile to avoid creating a new one named "base"
+        let activeBaseProfile = params.arguments[`${this.mProfileType}-profile`] || config?.properties?.defaults?.[this.mProfileType];
+
         // Populate the config with base profile information
-        if (profileConfig.defaults.base == null && profileConfig.profiles.base == null) {
-            profileConfig.profiles.base = {
-                type: "base",
+        if (activeBaseProfile == null) {
+            profileConfig.profiles[this.mProfileType] = {
+                type: this.mProfileType,
                 properties: {
                     host: session.ISession.hostname,
                     port: session.ISession.port,
                     rejectUnauthorized: session.ISession.rejectUnauthorized
                 },
                 secure: []
             };
-            profileConfig.defaults.base = "base";
+            profileConfig.defaults[this.mProfileType] = this.mProfileType;
+            activeBaseProfile = this.mProfileType;
+        } else {
+            lodash.set(profileConfig, config.api.profiles.getProfilePathFromName(activeBaseProfile), {
+                type: this.mProfileType,
+                properties: {
+                    ...config.api.profiles.get(activeBaseProfile),
+                    host: session.ISession.hostname ?? params.arguments.host,
+                    port: session.ISession.port ?? params.arguments.port,
+                    rejectUnauthorized: session.ISession.rejectUnauthorized ?? params.arguments.rejectUnauthorized
+                },
+                secure: []
+            });
+        }
```

Comment:
I'm afraid this is just to keep the command behaving the same way as before.
in the scenario where there was no previous config, we were not storing these properties.
And when there was an existing base profile... the properties would have been properly set in the `ISession` object. 

I could remove the `?? params.arguments...` since I do not believe this will ever be needed 😋